synapse/config/tls.py

```diff
@@ -299,12 +299,6 @@ def read_tls_certificate(self) -> crypto.X509:
         cert_pem = self.read_file(cert_path, "tls_certificate_path")
         cert = crypto.load_certificate(crypto.FILETYPE_PEM, cert_pem)
 
-        # Check if it is self-signed, and issue a warning if so.
```

Comment:
there's some incorrect text in the docstring which needs an update now.

Reply:
Oh oops